ycb_item:
  name: "cracker"
  x: 16.403600692749023
  y: 21.343700408935547
  z: 7.179999828338623

# ycb_item:
#   name: sugar
#   x: 9.2677001953125
#   y: 17.62529945373535
#   z: 4.513400077819824

# ycb_item:
#   name: spam
#   x: 10.16469955444336
#   y: 8.354299545288086
#   z: 5.7600998878479
